app/validators/typoed_email_validator.rb

```diff
@@ -20,13 +20,16 @@ class TypoedEmailValidator < ActiveModel::EachValidator
       gmaile.com
       gmaill.com
       gmali.com
+      gnail.com
       hotamil.com
       hotmai.com
       hotmaill.com
       iclooud.com
       iclould.com
       icluod.com
+      xn--gmail-xk1c.com
```

Comment:
I have no idea how someone managed to sign up with that.  It could be that browser seems to do some substitution in the domain...

Firefox 103 doesn't do the substitution, Safari 15.6 doesn't allow emoji in email (it asks you to "Enter an email address"), and other browsers do not exist on my machines to test.  So what the hell.